osv/schema.json

```diff
@@ -0,0 +1 @@
+../osv-schema/validation/schema.json
```

Comment:
Can we just include osv-schema in the "osv" directory instead to avoid the "../" ?